tests/test_utils.py

```diff
@@ -55,22 +55,26 @@ def test_write_labels():
     assert(os.path.exists(file_path))
 
 
-def test_balance():
-    unbalanced_tuples = [
-        ("data1", 1),
-        ("data2", 1),
-        ("data3", 0),
-        ("data4", 0),
-        ("data5", 0),
-        ("data6", 1)]
+data = [
```

Comment:
Nit: Call this `unbalanced_data` or `test_balance_data` as `data` is too generic.